### PR TITLE
Proofread readme.md

### PR DESCRIPTION
**Pull request by mockingjamie**
Made minor phrasing and grammatical improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Channel: #refugerestrooms
 
 Copyright (C) 2014–2016 Teagan Widmer and contributors
 
-This program is free software; you can redistribute it and/or modify
+This program is free software; you can redistribute and/or modify
```
Comment (backspace) [inline README.md]: The `it` is on the next line for this, I think???

**Comment by tkwidmer** [comment]
this lgtm

**Comment by tkwidmer** [comment]
Thanks for helping out @mockingjamie

File: README.md (hunk 1 above)
Comment (tkwidmer) [inline README.md]: there are two it's in the previous version, and only one is needed grammatically i think.

**Pull request merged**